client/src/components/TinymceInlineToolbar/TinymceInlineToolbar.js

```diff
@@ -0,0 +1,243 @@
+/* global window, document, tinymce */
+
+/**
+ * Inline / floating tinymce toolbar
+ */
+class TinymceInlineToolbar {
+
+  /**
+   * @param  {Object} tinymce editor instance
+   * @param  {Array} array of button literal object
+   */
+  constructor(editor, buttons) {
+    this.mceIframe = document.getElementById(`${editor.id}_ifr`);
+    this.container = editor.getContainer();
+    this.mceToolbar = null;
+    this.mceStatusbar = null;
+
+    if (this.container) {
+      this.mceToolbar = tinymce.$('.mce-toolbar-grp', this.container)[0];
+      this.mceStatusbar = tinymce.$('.mce-statusbar', this.container)[0];
```

Comment:
we could also use `this.container.querySelector('.className');` which selects the first instance element without coupling to jQuery.
I didn't know jQuery was a part of tinymce.